### PR TITLE
Improve the layout and info of the oauth popup

### PR DESCRIPTION
**Pull request by rodrigok**
captura de tela 2016-01-13 as 17 33 59

### CODE REVIEW

File: i18n/en.i18n.json [hunk 1]
```diff
@@ -588,6 +590,7 @@
   "Yes_mute_user" : "Yes, mute user!",
   "Yes_remove_user" : "Yes, remove user!",
   "you_are_in_preview_mode_of" : "You are in preview mode of channel #<strong>__room_name__</strong>",
+  "You_are_logged_as" : "You are logged as",
```
Comment (sampaiodiego) [inline i18n/en.i18n.json]: I think the correct message should be `You are logged in as` but not sure :stuck_out_tongue:

**Comment by sampaiodiego** [comment]
lgtm

**Comment by rodrigok** [comment]
LGTM

**Comment by engelgabriel** [comment]
lgtm

**Pull request merged**